### PR TITLE
Upstream Train sync 2021-01-11

### PR DESCRIPTION
**Pull request by paulbrowne**

### CODE REVIEW

File: ansible/roles/octavia/templates/octavia-health-manager.json.j2 [hunk 1]
```diff
@@ -1,5 +1,5 @@
-{
-    "command": "octavia-health-manager --config-file /etc/octavia/octavia.conf",
+
+mmand": "octavia-health-manager --config-file /etc/octavia/octavia.conf",
```
Comment (priteau) [inline ansible/roles/octavia/templates/octavia-health-manager.json.j2]: There was an issue during merging here.

**Review by priteau — CHANGES_REQUESTED** [review]

File: ansible/roles/octavia/templates/octavia-health-manager.json.j2 (hunk 1 above)
Comment (paulbrowne) [reply ansible/roles/octavia/templates/octavia-health-manager.json.j2]: Thanks for fixing that one

**Comment by brtkwr** [comment]
I don't think this branch contains all the latest changes.

**Comment by brtkwr** [comment]
This doesnt contain the tip of https://github.com/openstack/kolla-ansible/tree/stable/train

**Comment by priteau** [comment]
> This doesnt contain the tip of https://github.com/openstack/kolla-ansible/tree/stable/train

It was merged today after I pushed my branch.

**Review by priteau — APPROVED** [review]

**Pull request merged**